lib/northstar-endpoint.js

```diff
@@ -10,6 +12,47 @@ class NorthstarEndpoint {
     this.client = client;
   }
 
+  /**
+   * Helper function to execute simple get.
+   */
+  executeGet(endpoint, query) {
```

Comment:
@sergii-tkachenko I was curious about passing `endpoint` here -- in theory, wouldn't it always be `this.endpoint` ?